src/Projection/PdoEventStoreReadModelProjector.php

```diff
@@ -151,6 +161,7 @@ public function __construct(
         $this->persistBlockSize = $persistBlockSize;
         $this->sleep = $sleep;
         $this->status = ProjectionStatus::IDLE();
+        $this->triggerPcntlSignalDispatch = $triggerPcntlSignalDispatch && extension_loaded('pcntl');
```

Comment:
I think this would limit portability of projections across systems (e.g. when a library defines a projection with this flag enabled, while the user's system doesn't have it enabled). Personally I prefer the degraded functionality approach, but it's your call.